### ISSUE TITLE
angular: should definitely exist some day

### ISSUE THREAD

**Issue by ryanfiller** [ryanfiller/color-contrast-table#3]

**Comment by atlemagnussen** [comment]
If you create a web component / custom element version it usually works on all frameworks

**Comment by ryanfiller** [comment]
You are correct, I believe there is a way to get a web component working in any framework (although I think React can get weird with putting props in the DOM?). I haven't ever worked with web components, but I was thinking about doing a learning-type project with them soon. This might be a good place to start, thank you for the idea!

**Comment by atlemagnussen** [comment]
np! I use web components with Svelte and the old AngularJs with no special handling, it just actually works, so I figured it works everywhere, but I haven' tested React
Writing a pure custom element with no dependencies was weird and I avoided them a long time. Attributes vs properties and rendering is tedious to boilerplate. But when I discovered the micro-framework lit.dev these problems went away :)
